src/matlab2tikz.m

```diff
@@ -987,9 +1022,8 @@ case guitypes()
         % Take only the first child from a pure hggroup (e.g. bodeplots)
         for ii = 1:numel(entries)
             entry     = entries(ii);
-            isHggroup = any(strcmpi(get(entry,'Type'),{'hggroup', 'matlab.graphics.primitive.Group'})) & ...
-                            strcmpi(class(handle(entry)),'hggroup');
-            if isHggroup
+            isHggroupClass = strcmpi(class(handle(entry)),'hggroup');
+            if isHggroupClass
```

Comment:
:+1:. However, you might want to add a short note to state that `handle` is not supported on Octave (I know this is in a MATLAB-only branch and so it shouldn't be a problem).